### PR TITLE
Create 2019-08-08-2019-08-08.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
Second lab Meeting Blog. Thanks everyone who came!!! Was really interesting and helpful again. Comments, suggestions, improvement welcome :)

### CODE REVIEW

File: _posts/blog/2019-08-08-2019-08-08-lab-meeting.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+**Louise** shared one on Traffic mobility.
+It's currently a private repo but it will go public at some point.
+It contains the `docker-compose` file that sets up the traffic visualisation project Louise showed in the slides for a previous lab meeting (see blog for the 25th July):
+https://whitakerlab.github.io/blog/2019-07-25-lab-meeting/
+Louise's README is also the place where her and her team drop any useful documents from meetings and planning sessions.
+This README is all about the general: what you should do while you're here, and how to get it up and running. 
+
+**Sarah** shared her README: [binderhub-deploy button](https://github.com/alan-turing-institute/binderhub-deploy/blob/master/README.md). 
+This ✨automagically ✨deploys a BinderHub to Azure using a button click!
+This README is basically about how you use the software. T
```
Comment (sgibson91) [inline _posts/blog/2019-08-08-2019-08-08-lab-meeting.md]:
```suggestion
This README is basically about how you use the software.
```

File: _posts/blog/2019-08-08-2019-08-08-lab-meeting.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+here are three ways you can run the repository, so it goes through all three ways of doing it.
```
Comment (sgibson91) [inline _posts/blog/2019-08-08-2019-08-08-lab-meeting.md]:
```suggestion
There are three ways you can run the repository, so it goes through all three ways of doing it. 
```

**Review by sgibson91 — APPROVED** [review]
A typo that got caught up in the new-sentence/new-line splitting. Otherwise looks great, thank you!

**Review by minnieho1115 — APPROVED** [review]

**Review by KirstieJane — APPROVED** [review]
Heya! Thank you so much @GeorgiaHCA for writing this up!!

I've trimmed a little of the detail and fixed some links.

If it looks good to you let me know!

**Comment by GeorgiaHCA** [comment]
Looks good to me! Thanks for checking it over - go ahead and merge :)

**Pull request merged**